RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Fix resolvers imported from another modules
```

Comment:
Can we add a bit more detail for other readers on what was broken (and is now fixed)

Reply:
Yes. As the tests show the removed line did not only break fields with a resolver from a different module but also inheritance. I'm looking into that right now.